qtopconf.yaml

```diff
@@ -22,6 +22,8 @@ schedulers:
     oarstat_file: %(savepath)s/oarstat%(pid)s.txt, oarstat
   sge:
     sge_file_stat: %(savepath)s/qstat%(pid)s.F.xml.stdout, qstat -F -xml -u '*'
+  demo:
```

Comment:
Would be nice if all those ware going back to individual sources with a mechanism similar to `get_mnemonic():`.